_posts/2018-04-17-libra.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Libra
+date: 2018-04-17
+categories:
+    - project
+    - Flask
+    - MySQL
+    - Flask_SQLAlchemy
+    - Python
+description: web knowledgebase where users to input skills and view skills of other users
```

Comment:
Confusing description; "knowledgebase" should be two words; desc. should read more like a sentence

"A web knowledge base where users may input their skills and view the skills of others."